mava/components/training/losses.py

```diff
@@ -29,12 +29,28 @@
 from mava.core_jax import SystemTrainer
 
 
-class DefaultValueLossFunction(Component):
+class ValueLoss(Component):
+    @abc.abstractmethod
+    def on_training_utility_fns(self, trainer: SystemTrainer) -> None:
+        """An abstract class for a component that defines the
+        final value loss function."""
+
+    @staticmethod
+    def name() -> str:
+        """_summary_
+
+        Returns:
+            _description_
+        """
+        return "value_loss"
+
+
+class SquaredErrorLoss(ValueLoss):
```

Comment:
Sorry, I know I suggest the original name change, and this is nitpicky, but should we change it to `SquaredErrorValueLoss` to be more consistent?

Reply:
Or change `HuberValueLoss` to `HuberLoss`

Reply:
Okay I will change the Square error to `SquaredErrorValueLoss`

Reply:
just we need to wait for the benchmark to end

Reply:
Done!